api/valor_api/backend/metrics/classification.py

```diff
@@ -235,38 +372,8 @@ def _compute_confusion_matrix_at_grouper_key(
         that have both a ground truth and prediction with label key `label_key`. Otherwise
         returns the confusion matrix.
     """
-    label_key_filter = list(
-        grouper_mappings["grouper_key_to_label_keys_mapping"][grouper_key]
-    )
 
-    # groundtruths filter
-    gFilter = groundtruth_filter.model_copy()
```

Comment:
I moved this section out of this function and into the preceding scope so that I could use these objects for `_compute_curves`.